artworks/models.py

```diff
@@ -1,3 +1,87 @@
 from django.db import models
-
+from art_web_app.models import CustomModel
 # Create your models here.
+from django.contrib.auth.models import User
+from io import BytesIO
+from PIL import Image
+from django.core.files.base import ContentFile
+from utils.constants import THUMB_SIZE
+import os
+
+
+class Categories(CustomModel):
```

Comment:
Please keep model names to singular, e.g. `Category`. I know I've named some with plural last course. My bad :))